src/temperature.jl

```diff
@@ -0,0 +1,17 @@
+"""
+Returns the temperature unchanged.
+# Arguments
+- `current_temperature::Float`: the current temperature of the search.
+"""
+function const_temperature(current_temperature)
+    return current_temperature
+end
+
+"""
+Returns the temperature decreased by 1%.
+# Arguments
+- `current_temperature::Float`: the current temperature of the search.
+"""
+function decreasing_temperature(current_temperature)
+    return 0.99 * current_temperature
```

Comment:
Maybe 0.99 should be a configurable parameter?

Reply:
Ah, I missed that the temperature function is something the user provides. In that case I think the current implementation is fine.

Reply:
Right now the implementation does not allow this. 
We could add a new function `construct_stochastic_enumerator` that lets the user provide their own implementations of `accept` `propose` `temperature` `neighbourhood`